### PR TITLE
Feat: Readme Updates [MTT-5460]

### PR DESCRIPTION
**Pull request by jilfranco-unity**
### Description
Updates the dynamic prefab sample readme, as well as a refresh for our global bitesize samples readme

!!!!! The dynamic prefabs readme is not done, please don't review yet

### Issue Number(s)
[jira ticket MTT-5460 here](https://jira.unity3d.com/browse/MTT-5460)

### Contribution checklist
 - [ ] Tests have been added for the project and/or any internal package
 - [ ] Release notes have been added to the [project changelog](../CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

File: Basic/DyanmicPrefabs/README.md [hunk 1]
```diff
@@ -1,27 +1,38 @@
...
+
+- Adding a prefab on the server **does not** automatically add it on the client - it's up to you
   to make sure the client and server are synchronized via whatever method makes sense for your game
-  (RPCs, configs, deterministic loading, etc)
-- If the server sends a Spawn message to a client that has not yet added a prefab for, the spawn message
-  and any other relevant messages will be held for a configurable time (default 1 second, configured via
-  NetworkConfig.SpawnTimeout) before an error is logged. This is intented to enable the SDK to gracefully
-  handle unexpected conditions (slow disks, slow network, etc) that slow down asset loading. This timeout
-  should not be relied on and code shouldn't be written around it - your code should be written so that
+  (RPCs, configs, deterministic loading, etc).
```
Comment (SamuelBellomo) [inline Basic/DyanmicPrefabs/README.md:17]:
```suggestion
  (RPCs, configs, deterministic loading, etc). This sample lists some of those possible methods with basic implementations you can adapt to your various use cases.
```

File: Basic/DyanmicPrefabs/README.md [hunk 2]
```diff
@@ -1,27 +1,38 @@
...
+
+- If the server sends a Spawn message to a client that does not yet have the corresponding prefab loaded, the spawn message
+  (and any other relevant messages) will be held for a configurable time before an error is logged (default 1 second, configured via
+  NetworkConfig.SpawnTimeout). This is intented to enable the SDK to gracefully
+  handle unexpected conditions that slow down asset loading (slow disks, slow network, etc). This timeout
+  should not be relied upon and code shouldn't be written around it - your code should be written so that
   the asset is expected to be loaded before it's needed.
-- Currently it's impossible to latejoin for clients after a dynamic prefab has been spawned by the server. The reason for this is that the initial sync doesn't allow us any time to load prefabs that are yet absent on the client.
+  
+- It's currently impossible for clients to late join after a dynamic prefab has been spawned by the server - this is because the initial sync doesn't allow us any time to load prefabs that are aren't yet loaded on the client.
```
Comment (SamuelBellomo) [inline Basic/DyanmicPrefabs/README.md:26]: This isn't true anymore, no?

File: Basic/DyanmicPrefabs/README.md [hunk 3]
```diff
@@ -1,27 +1,38 @@
...
+<br><br>
+
+Therefore, based on these limitations, the following use-cases are covered in this sample:
+- PreloadingSample.cs - this is the simplest case of a dynamic prefab - we just add it to the list of prefabs before we connect on all the peers.
+
+- AppController.cs and DynamicPrefabManager.cs - this is a more complex case where the server spawns things that the client haven't yet loaded. There are two loading strategies - one is "synchronous" spawning, which ensures that all clients have acknowledged that they have loaded that prefab before the server actually spawns the NetworkObject. The other one is spawning the object immediately and using visibility system to hide that object from clients that haven't loaded the prefab yet (when they do acknowledge the prefab loading - the server will show the relevant hidden objects to the client). This sample also shows how we could handle the issue with initial synchronization breaking for late joiners by using some custom logic in connection approval delegate.
+<br><br>
```
Comment (SamuelBellomo) [inline Basic/DyanmicPrefabs/README.md:33]: This needs updating

**Comment by jilfranco-unity** [comment]
closing to make a new PR off of develop post folder structure update to avoid messy merging

**Pull request closed without merging**